pkg/cpi/commonUtils.go

```diff
@@ -21,6 +23,31 @@ type TokenParameters struct {
 	Client                       piperhttp.Sender
 }
 
+// CpiServiceKey contains information about a CPI service key
+type ServiceKey struct {
+	OAuth OAuth `json:"oauth"`
+}
+
+type OAuth struct {
```

Comment:
Add short docu comment above.

Reply:
Okay, what should I add for the comment? Just that we're using service key instead of the other 4 parameters? And should I change this `ServiceKey` to be `APIServiceKey `as well?